### PR TITLE
fix typos and examples for test cases

### PR DESCRIPTION
**Pull request by AakashKumarNain**
The syntax for the test cases wasn't correct. Added more information regarding the implementation as well.

### CODE REVIEW

**Comment by bhack** [comment]
Was this derived from https://twitter.com/fchollet/status/1280730971717820416?

**Comment by AakashKumarNain** [comment]
Thanks for the review @WindQAQ . In the earlier version. I kept it that way only and I agree that it is better to use that way. But if you look at the docs for `metrics` in TF core, for example [categorical accuracy](https://www.tensorflow.org/versions/r2.3/api_docs/python/tf/keras/metrics/CategoricalAccuracy), they explicitly show `update_state()`. I thought it would be better to mention it here that way as well so that there is no confusion at all but we can take a call on which version we want to go with

**Comment by AakashKumarNain** [comment]
@bhack nah, not from that.

**Comment by bhack** [comment]
> @bhack nah, not from that.

Ok cause point 2. is that list.
